docs/en/setup/backend/backend-alarm.md

```diff
@@ -41,10 +43,16 @@ Such as in **percentile**, `value1` is threshold of P50, and `-, -, value3, valu
 backend deployment env time.
 - **Count**. In the period window, if the number of **value**s over threshold(by OP), reaches count, alarm
 should send.
+- **Only as condition**. Specify if the rule can send notfiy or just as an conditon of composite rule
 - **Silence period**. After alarm is triggered in Time-N, then keep silence in the **TN -> TN + period**.
 By default, it is as same as **Period**, which means in a period, same alarm(same ID in same 
 metrics name) will be trigger once. 
 
+### Composite rules
+Composite rule is constituted by following keys
+- **Rule name**. Unique name, show in alarm message. Must end with `_rule`.
+- **Expression**. Specify how to compose rules, support and/or expression
```

Comment:
Please test the `parenthesis` case, this is important with `and` and `or`.

Reply:
added test case